torch/fx/experimental/fx2trt/fx2trt.py

```diff
@@ -174,6 +175,7 @@ def run(
             cache = builder_config.create_timing_cache(b"")
         builder_config.set_timing_cache(cache, False)
 
+        builder_config.profiling_verbosity = profiling_verbosity if profiling_verbosity else trt.ProfilingVerbosity.LAYER_NAMES_ONLY
```

Comment:
looks like this failed in older tensorrt versions, should we add an assert for tensorrt version?

Reply:
cc @842974287 as well